src/java.base/share/classes/jdk/internal/module/ModulePatcher.java

```diff
@@ -131,14 +131,15 @@ public ModuleReference patchIfNeeded(ModuleReference mref) {
 
                     // exploded directory without following sym links
                     Path top = file;
-                    Files.find(top, Integer.MAX_VALUE,
-                               ((path, attrs) -> attrs.isRegularFile()))
-                            .filter(path -> (!isAutomatic
-                                    || path.toString().endsWith(".class"))
-                                    && !isHidden(path))
-                            .map(path -> toPackageName(top, path))
-                            .filter(Checks::isPackageName)
-                            .forEach(packages::add);
+                    try (Stream<Path> stream = Files.find(top, Integer.MAX_VALUE,
+                            ((path, attrs) -> attrs.isRegularFile()))) {
+                            stream.filter(path -> (!isAutomatic
+                                || path.toString().endsWith(".class"))
+                                && !isHidden(path))
+                                .map(path -> toPackageName(top, path))
+                                .filter(Checks::isPackageName)
+                                .forEach(packages::add);
```

Comment:
The formatting seems to a bit messed up here too, I don't know if you means to do that. In the try-block the indentation of stream.filter should be 4 spaces, not 8. Also can you restore the alignment of the expression provided to filter then it will be easier to read.